crowbar_framework/app/views/installer/upgrades/_nav.html.haml

```diff
@@ -0,0 +1,60 @@
+%ul.nav.nav-stacked.wizard-nav.col-lg-3
+  %li{ class: active == :prepare ? "active" : nil }
+    %a
+      .pull-right
+        = icon_tag "chevron-right"
+      = t(".prepare")
+
+  %li{ class: active == :download ? "active" : nil }
+    %a
+      .pull-right
+        = icon_tag "chevron-right"
+      = t(".download")
+
+  %li{ class: active == :confirm ? "active" : nil }
+    %a
+      .pull-right
+        = icon_tag "chevron-right"
+      = t(".confirm")
+
+  %li{ class: active == :start ? "active" : nil }
```

Comment:
just `%li` because this never can reached this is also valid for the flowing li's